frontend/src/App.css

```diff
@@ -0,0 +1,38 @@
+.App {
```

Comment:
- [ ] The mobile version has a lot of overflowing when I got 3 doctors:

![image](https://user-images.githubusercontent.com/57421823/199940194-6fe1fd98-5fea-47f1-8a93-ba1c2a39f1fd.png)

![image](https://user-images.githubusercontent.com/57421823/199940234-2d3822fa-5fe2-4d29-9564-2aa537dcdcde.png)

In the last screenshot, for instance, I can't open de nav.


Kindly retouch your mobile version, so it's more usable and avoid element overflowing  




That way, your project will look beautiful 💪